app/src/main/java/org/fossasia/susi/ai/skills/SkillsActivity.kt

```diff
@@ -111,10 +116,9 @@ class SkillsActivity : AppCompatActivity() {
 
             action!!.setDisplayShowCustomEnabled(false) //disable a custom view inside the actionbar
             action.setDisplayShowTitleEnabled(true) //show the title in the action bar
-
             //add the search icon in the action bar
             mSearchAction?.icon = resources.getDrawable(R.drawable.ic_open_search)
-
+            hideKeyboard()
```

Comment:
actually keyboard should not be hidden in the skill listing activity on clicking the cross button

Reply:
handle this by hiding the keyboard on opening the about, settings and skill details and instead of adding handleMenuSearch() in these cases make another function that does this

Reply:
@batbrain7 changes are added, can you check?

Reply:
@agh372 no changes seen